src/main/java/org/jenkinsci/plugins/junitrealtimetestreporter/RealtimeJUnitStep.java

```diff
@@ -143,7 +144,7 @@ public StepExecution start(StepContext context) throws Exception {
     }
 
     @SuppressFBWarnings("SE_BAD_FIELD") // FIXME
-    static class Execution extends StepExecution {
+    static class Execution extends GeneralNonBlockingStepExecution {
```

Comment:
And this will also break serialization across upgrades. See https://github.com/jenkinsci/workflow-basic-steps-plugin/pull/78#discussion_r247249283 for an example.